config/environments/development.rb

```diff
@@ -58,5 +58,7 @@
   # Allow disabling the webpack dev server from the settings
   config.webpack.dev_server.enabled = SETTINGS.fetch(:webpack_dev_server, true)
   config.webpack.dev_server.https = SETTINGS.fetch(:webpack_dev_server_https, false)
-  config.hosts << SETTINGS[:fqdn]
+
+  config.hosts += SETTINGS[:hosts]
+  config.hosts += ENV['RAILS_DEVELOPMENT_HOSTS'].split(',')
```

Comment:
I tested this, and it seems that the environment variable isn't automatically added by Rails to your `config.hosts` - it only works if you explicitly add it like this.